include/mbedtls/platform_time.h

```diff
@@ -39,6 +39,22 @@ typedef MBEDTLS_PLATFORM_TIME_TYPE_MACRO mbedtls_time_t;
 typedef time_t mbedtls_time_t;
 #endif /* MBEDTLS_PLATFORM_TIME_TYPE_MACRO */
 
+#if defined(MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO)
+typedef MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO mbedtls_ms_time_t;
+#else
+typedef signed long long mbedtls_ms_time_t;
+#endif /* MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO */
+
+/**
+ * \brief Get time in milliseconds.
+ *
+ * \return Current time in milliseconds
```

Comment:
This should really include something that since "since WHEN", where WHEN can be 00:00 1 Jan 1970 GMT (typical for Unixes) but is different on Windows (January 1, 1601 according to old notes of mine, but should be checked) - and we should state if  this is going to be monotonically increasing or if it's a wallclock value (which can go backwards, e.g. during change of daylight savings)

Reply:
Understand.  And I will add some note about it.

That is used for time difference. I think we should not consider about `since WHEN` here. But we should add warning here.